.github/workflows/ci.yml

```diff
@@ -0,0 +1,82 @@
+name: CI
+
+on:
+    push:
+        branches:
+            - 'release/**'
+    pull_request:
+
+env:
+    ALIAS_VERSION: '7.4'
+    VERSION_FOR_ALIAS: '7.4.11'
+
+jobs:
+    test:
+        name: Build PHP ${{ matrix.php-version }} (${{ matrix.type }})
+
+        runs-on: ubuntu-latest
+
+        continue-on-error: true
+
+        strategy:
+            matrix:
+                php-version:
+                    - '7.4.11'
+                    - '7.4.10'
+#                    - '7.4.9'
+#                    - '7.4.8'
+#                    - '7.4.7'
+#                    - '7.4.6'
+#                    - '7.4.5'
+#                    - '7.4.4'
+#                    - '7.4.3'
+#                    - '7.4.2'
+#                    - '7.4.1'
+                type: [ 'standard', 'alpine' ]
+
+        steps:
+            -   name: "Checkout code"
+                uses: actions/checkout@v2
+
+            -   if: matrix.type == 'alpine'
+                run: ./build_alpine_version.sh ${{ matrix.php-version }}
+
+            -   if: matrix.type == 'alpine'
+                run: docker run --rm -t exozet/php-fpm:${{ matrix.php-version }}-alpine php -i
```

Comment:
```diff
-                run: docker run --rm -it exozet/php-fpm:${{ matrix.php-version }}-alpine php -i
+                run: docker run --rm -t exozet/php-fpm:${{ matrix.php-version }}-alpine php -i
```

@DracoBlue I had to change `-it` to `-t`, otherwise I got an error. Is this ok?

same for the other